src/player.cpp

```diff
@@ -95,8 +95,7 @@ void game::init_morale()
     _("Killed Friend"),
     _("Guilty about Killing"),
     _("Guilty about Mutilating Corpse"),
-    _("Chimerical Mutation"),
-    _("Fey Mutation"),
+    _("Mutation"),
```

Comment:
I don't mind if statements, but then I'm not opposed to mutations being C++; this still needs C++ to handle effects, anyway.